scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3851,368 @@ def rvs(self, dim, size=1, random_state=None):
 
 
 unitary_group = unitary_group_gen()
+
+
+_mvt_doc_default_callparams = \
+"""
+loc : array_like, optional
+    Location of the distribution. (default ``0``)
+shape : array_like, optional
+    Positive definite matrix of the distribution. (default ``1``)
+df : integer, optional
+    Degrees of freedom of the distribution; must be a positive whole number. (default ``1``) 
+allow_singular : bool, optional
+    Whether to allow a singular matrix. (default ``False``)
+"""
+
+_mvt_doc_callparams_note = \
+"""Setting the parameter `loc` to ``None`` is equivalent to having `loc`
+be the zero-vector. The parameter `shape` can be a scalar, in which case
+the shape matrix is the identity times that value, a vector of
+diagonal entries for the shape matrix, or a two-dimensional array_like.
+"""
+
+_mvt_doc_frozen_callparams_note = \
+"""See class definition for a detailed description of parameters."""
+
+mvt_docdict_params = {
+    '_mvt_doc_default_callparams': _mvt_doc_default_callparams,
+    '_mvt_doc_callparams_note': _mvt_doc_callparams_note,
+    '_doc_random_state': _doc_random_state
+}
+
+mvt_docdict_noparams = {
+    '_mvt_doc_default_callparams': "",
+    '_mvt_doc_callparams_note': _mvt_doc_frozen_callparams_note,
+    '_doc_random_state': _doc_random_state
+}
+
+
+class multivariate_t_gen(multi_rv_generic):
+    r"""
+    A multivariate t-distributed random variable.
+    The `loc` parameter specifies the location. The `shape` parameter specifies
+    the positive definite shape matrix. The `df` parameter specifies the
+    degrees of freedom.
+    In addition to calling the methods below, the object itself may be called
+    as a function to fix the location, shape matrix, and degrees of freedom
+    parameters, returning a "frozen" multivariate t-distribution random.
+    .. versionadded:: 1.5.0
```

Comment:
This isn't rendering properly ([here](https://21176-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.stats.multivariate_normal.html#scipy.stats.multivariate_normal) is what it looks like currently). Please take a look at the source for some of the other distributions and add whitespace as they do. Unfortunately, Sphinx is very particular about that sort of thing.

Reply:
Can we resolve this? The docs render properly for me, I assume because of @rlucas7's changes [here](https://github.com/scipy/scipy/pull/12582#issuecomment-691762038).

Reply:
> Can we resolve this? The docs render properly for me, I assume because of @rlucas7's changes [here](https://github.com/scipy/scipy/pull/12582#issuecomment-691762038).

Sometimes it takes github (browser application) a little while to update the client rendering with diff changes. I wasn't seeing the "outdated" note before, see it now though and agree the changes are taken care of, marking resolved.